documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -11,13 +11,28 @@ This whitepaper is an introduction to writing a Display Serial Interface (DSI) d
 
 The DSI display is designed to work with all models of Raspberry Pi, however early models that do not have mounting holes (the Raspberry Pi 1 Model A and B) will require additional mounting hardware to fit the HAT-dimensioned stand-offs on the display PCB.
 
-=== Physical Installation
+=== Connecting a Raspberry Pi
 
-The following image shows how to attach the Raspberry Pi to the back of the Touch Display (if required), and how to connect both the data (ribbon cable) and power (red/black wires) from the Raspberry Pi to the display. If you are not attaching the Raspberry Pi to the back of the display, take extra care when attaching the ribbon cable to ensure it is the correct way round. The black and red power wires should be attached to the GND and 5v pins respectively.
+The following image shows how to attach a Raspberry Pi to the back of the Touch Display using the included standoffs. It also shows the connections you need to make between Raspberry Pi and the display's PCB. You can also mount the touch display in a separate chassis if you have one available. The connections remain the same, though you may need longer cables depending on the chassis you use.
 
-image::images/GPIO_power-500x333.jpg[DSI Display Connections]
+image::images/GPIO_power-500x333.jpg[DSI Display pinout]
 
-The other three pins should be left disconnected, unless connecting the display to an original Raspberry Pi 1 Model A or B. See the section on xref:display.adoc#legacy-support[legacy support] for more information on connecting the display to an original Raspberry Pi.
+Connect one end of the ribbon cable to the `RPI-DISPLAY` port on the Touch Display PCB (make sure the blue stripe faces the PCB) and then connect the other end to the `DISPLAY` port on the Raspberry Pi (the blue stripe should face outwards).
+
+==== Powering the Touch Display
+
+There are two options to power the Touch Display: using the Raspberry Pi's GPIO to send power, or powering the display directly with a separate micro USB cable. 
```

Comment:
Should sub-section, if there are two options there should be two separate sections.